stl/inc/filesystem

```diff
@@ -2453,6 +2469,11 @@ namespace filesystem {
             return _Path == _Rhs._Path;
         }
 
+#if _HAS_CXX20
+        _NODISCARD strong_ordering operator<=>(const directory_entry& _Rhs) const noexcept {
+            return _Path <=> _Rhs._Path;
+        }
+#else // ^^^ _HAS_CXX20 / !_HAS_CXX20 vvv

```

Comment:
Shouldn't we move `operator<` from line 2464 into the `#else` branch of this preprocessor conditional as well? (-1 to whoever put the operators in this non-standard order in the first place.)